pkg/cmd/roachprod/flags.go

```diff
@@ -410,7 +416,7 @@ func initFlags() {
 		cmd.Flags().BoolVar(&urlOpen, "open", false, "Open the url in a browser")
 	}
 
-	for _, cmd := range []*cobra.Command{createCmd, listCmd, destroyCmd, extendCmd, logsCmd} {
+	for _, cmd := range []*cobra.Command{createCmd} {
```

Comment:
Won't this break listing or destroying clusters for specific users (not the VM user, but the creator of the cluster)?